core/src/main/java/com/datastax/dse/driver/internal/core/graph/DseGraphTraversal.java

```diff
@@ -33,16 +33,6 @@ public DseGraphTraversal(AsyncGraphResultSet firstPage) {
     this.graphNodeIterator = GraphResultSets.toSync(firstPage).iterator();
   }
 
-  @Override
-  @SuppressWarnings("deprecation")
-  public org.apache.tinkerpop.gremlin.process.remote.traversal.RemoteTraversalSideEffects
-      getSideEffects() {
-    // This was deprecated as part of TINKERPOP-2265
-    // and is no longer being promoted as a feature.
-    // return null but do not throw "NotSupportedException"
-    return null;
-  }
-
```

Comment:
[Officially removed](https://tinkerpop.apache.org/docs/current/upgrade/#_deprecation_removal) in Tinkerpop 3.5.0